src/main/java/com/BrewMES/demo/API/BrewMESController.java

```diff
@@ -92,4 +94,20 @@
 
         return response;
     }
+    //make this method return last 10 batches
+    @GetMapping(value = "/batches")
+    public ResponseEntity<Object> getBatches() {
+        return new ResponseEntity<>(new StringResponse("Not Implemented yet", HttpStatus.NOT_IMPLEMENTED.value()), HttpStatus.NOT_IMPLEMENTED);
+    }
+    //make this method return a batch based on it's id
+    @GetMapping(value = "/batches/{id}")
+    public ResponseEntity<Object> getBatch(@PathVariable("id") String id) {
+        return new ResponseEntity<>(new StringResponse("Not Implemented yet", HttpStatus.NOT_IMPLEMENTED.value()), HttpStatus.NOT_IMPLEMENTED);
+    }
+
+    //make this method handle the make report and return and return json with it's location
+    @GetMapping(value = "/bathes/{id}/geneate")
```

Comment:
generate, ikke geneate... ENGLISH MOTHERFUCKER... DO YOU SPEAK IT?!

Reply:
Speak, yes. 
Write, not so much

Reply:
https://github.com/Semester3-03/Java-application/pull/72/commits/9fd5402bd13ac009e9df73c420f1795e92d7245f